### PR TITLE
optimize of errorcode 

### PR DESCRIPTION
**Pull request by casionone**
### Brief change log

Optimize the use of error code classes and use enumeration classes uniformly
Chinese error description needs to be adjusted to English ("Chinese") mode

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-commons/linkis-scheduler/src/main/scala/org/apache/linkis/scheduler/AbstractScheduler.scala [hunk 1]
```diff
@@ -39,8 +40,8 @@ abstract class AbstractScheduler extends Scheduler {
           .contains(EVENT_ID_SPLIT) || eventId.startsWith(EVENT_ID_SPLIT)
     ) {
       throw new SchedulerErrorException(
-        12011,
-        s"Unrecognized execId $eventId.（不能识别的execId $eventId.)"
+        UNRECOGNIZED_EXECID.getErrorCode,
+        UNRECOGNIZED_EXECID.getErrorDesc + s"$eventId."
```
Comment (casionone) [inline linkis-commons/linkis-scheduler/src/main/scala/org/apache/linkis/scheduler/AbstractScheduler.scala:44]: MessageFormat.format

File: linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/loader/loaders/DefaultEngineConnPluginLoader.java [hunk 2]
```diff
@@ -217,7 +219,7 @@ private EngineConnPlugin loadEngineConnPlugin(
       final Constructor<?>[] constructors = pluginClass.getConstructors();
       if (constructors.length == 0) {
         throw new EngineConnPluginLoadException(
-            "No public constructor in pluginClass [" + pluginClass.getName() + "]", null);
+            NO_PUBLIC_CONSTRUCTOR.getErrorDesc() + " [" + pluginClass.getName() + "]", null);
```
Comment (casionone) [inline linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/loader/loaders/DefaultEngineConnPluginLoader.java:222]: no need to adjust this

File: linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/loader/loaders/resource/BmlEngineConnPluginResourceLoader.java [hunk 3]
```diff
@@ -140,7 +142,7 @@ private void downloadResource(
       }
       if (!parentFile.exists() || !parentFile.canWrite()) {
         throw new EngineConnPluginLoadResourceException(
-            "Have no write permission to directory:[" + parentFile.getAbsolutePath() + "]", null);
+            NO_WRITE_PERMISSION.getErrorDesc() + ":[" + parentFile.getAbsolutePath() + "]", null);
```
Comment (casionone) [inline linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/loader/loaders/resource/BmlEngineConnPluginResourceLoader.java:145]: no need to adjust this

File: linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/utils/JobHistoryHelper.scala [hunk 4]
```diff
@@ -155,7 +158,10 @@ object JobHistoryHelper extends Logging {
     } {
       case errorException: ErrorException => throw errorException
       case e: Exception =>
-        val e1 = JobHistoryFailedException(QUERY_TASKID_ERROR.getErrorDesc + s"$taskID")
+        val e1 =
+          JobHistoryFailedException(
```
Comment (casionone) [inline linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/utils/JobHistoryHelper.scala:162]: no need to adjust this

File: linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/manager/rm/service/impl/DefaultResourceManager.scala [hunk 5]
```diff
@@ -590,7 +595,7 @@ class DefaultResourceManager extends ResourceManager with Logging with Initializ
     if (labelResource.getCreateTime != null && usedResource.getCreateTime != null) {
       if (labelResource.getCreateTime.getTime > usedResource.getCreateTime.getTime) {
         throw new RMErrorException(
-          10022,
+          RESOURCE_LATER_CREATED.getErrorCode,
           s"no need to clear this labelResource, labelResource:${labelResource} created time is after than usedResource:${usedResource}" +
             s"无需清理该标签的资源,该标签资源的创建时间晚于已用资源的创建时间"
```
Comment (casionone) [inline linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/manager/rm/service/impl/DefaultResourceManager.scala:600]: MessageFormat.format

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**